cv/cv/src/cv_utils/camera_projection.py

```diff
@@ -23,51 +26,60 @@ def __init__(self):
             self.depth_values += [self.depth_map[key]]
         self.depth_values = np.array(self.depth_values)
 
+        camera_info = rospy.wait_for_message('/zed/zed_node/rgb/camera_info', CameraInfo)
 
-        '''
-        Raw parameters (found in /usr/local/zed/settings)
-        fx = 1400.13
-        fy = 1400.13
-        cx = 1188.06
-        cy = 645.894
+        # Update camera intrinsics to account for the resized new images
+        scale_x = 330.0/camera_info.width
```

Comment:
Nit: put named constants for 330.0 and 180.0

Reply:
Would it also be better if we casted `camera_info.width` to a float (since we are treating it as a float here).

The same deal with the height